packages/data/src/index.js

```diff
@@ -182,6 +184,11 @@ export const use = defaultRegistry.use;
 /**
  * Registers a standard `@wordpress/data` store definition.
  *
- * @param {import('./types').WPDataStore} store Store definition.
```

Comment:
@nosolosw – is there any way we could improve `@wordpress/docgen` to ignore the import part so we don't have to use `typedef` workaround?